krun/results.py

```diff
@@ -183,17 +191,18 @@ def strip_results(self, key_spec):
     def append_exec_measurements(self, key, measurements):
         """Unpacks a measurements dict into the Results instance"""
 
+        # Has already been check earlier, but let's be defensive.
```

Comment:
Though if it has been checked earlier, doing it a second time seems a bit pointless, unless it could have changed?